server/sonar-web/src/main/js/apps/project-admin/store/actions.js

```diff
@@ -70,6 +76,49 @@ export const setProjectProfile = (projectKey, oldKey, newKey) =>
       });
     };
 
+export const RECEIVE_GATES = 'RECEIVE_GATES';
+export const receiveGates = gates => ({
+  type: RECEIVE_GATES,
+  gates
+});
+
+export const RECEIVE_PROJECT_GATE = 'RECEIVE_PROJECT_GATE';
+export const receiveProjectGate = (projectKey, gate) => ({
+  type: RECEIVE_PROJECT_GATE,
+  projectKey,
+  gate
+});
+
+export const fetchProjectGate = projectKey => dispatch => {
+  Promise.all([
+    fetchQualityGates(),
+    getGateForProject(projectKey)
+  ]).then(responses => {
+    const [allGates, projectGate] = responses;
+    dispatch(receiveGates(allGates));
+    dispatch(receiveProjectGate(projectKey, projectGate));
+  });
+};
+
+export const SET_PROJECT_GATE = 'SET_PROJECT_GATE';
+const setProjectGateAction = (projectKey, gateId) => ({
+  type: SET_PROJECT_GATE,
+  projectKey,
+  gateId
+});
+
+export const setProjectGate = (projectKey, oldId, newId) => dispatch => {
+  console.log(oldId, newId);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS2228)